src/modules/Pagination/Pagination.scss

```diff
@@ -0,0 +1,63 @@
+@import 'index.scss';
+
+.j-pagination {
+  .Pagimagic-nav {
```

Comment:
is it possible to change this class in the component or avoid touching it at all? kind a goes away from the conventions we have.